### PR TITLE
Fix PR-Creating tools

### PR DESCRIPTION
**Pull request by alvaroaleman**
For https://issues.redhat.com/browse/DPTP-1852

### CODE REVIEW

**Review by stevekuznetsov — APPROVED** [review]

File: cmd/autoperibolossync/main.go [hunk 1]
```diff
@@ -144,7 +144,7 @@ func main() {
 		labelsToAdd = append(labelsToAdd, labels.Approved, labels.LGTM)
 	}
 
-	if err := bumper.UpdatePullRequestWithLabels(gc, githubOrg, githubRepo, title, description, o.githubLogin, remoteBranch, "master", true, labelsToAdd); err != nil {
+	if err := bumper.UpdatePullRequestWithLabels(gc, githubOrg, githubRepo, title, description, o.githubLogin+remoteBranch, "master", remoteBranch, true, labelsToAdd); err != nil {
```
Comment (petr-muller) [inline cmd/autoperibolossync/main.go:147]: SHouldnt' the `source` be `login:remoteBranch` instead of concatenation?

Comment (alvaroaleman) [reply cmd/autoperibolossync/main.go:147]: /hold 
yes

Comment (alvaroaleman) [reply cmd/autoperibolossync/main.go]: @petr-muller @stevekuznetsov updated, ptal
/hold cancel

**Review by petr-muller — APPROVED** [review]

**Review by droslean — APPROVED** [review]

**Pull request merged**